src/sensors/drivers/generic_analog.cpp

```diff
@@ -187,6 +187,8 @@ void GenericAnalogDriver::takeCalibrationBurstMeasurement()
       externalADC->convertEnabledChannels();
     }
     takeMeasurement();
+    // TODO: check for 0 values, report them, and either skip them for calibration, or halt calibration and notify that board may have electrical issues?
+    // Thoughts: reasons for 0 values? electrical issue, power failure, or are there scenarios where 0's are acceptable but still need to be skipped?
```

Comment:
A value that's at the absolute edge of the range of the sensor is always very very suspicious.  I don't think they should be accepted during calibration - though we also have to make sure that if we get only 0's we aren't eternally stuck in calibration mode.